config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  root "users#index"
+
+  get '/users/:author_id/posts' => 'posts#index', as: :user_posts
+  get '/users/:author_id/posts/:id' => 'posts#show', as: :user_post
+  get '/users' => 'users#index'
+  get '/users/:id' => 'users#show'
```

Comment:
- [OPTIONAL] You can consider writing routes this way
```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```
as it is much shorter and it's easier to read.

Reply:
Hi again @gemmen29! Could you give me a little hand? I want to implement your suggestion. It works great, however I am using author_id for my posts, which I can retrieve using GET. So, is there a way to do the same using your implementation?
<img width="1440" alt="Captura de Pantalla 2022-07-11 a la(s) 11 34 31 a  m" src="https://user-images.githubusercontent.com/75434179/178314281-4660710a-6216-4271-8e47-cdc629ae96a9.png">

<img width="1440" alt="Captura de Pantalla 2022-07-11 a la(s) 11 34 46 a  m" src="https://user-images.githubusercontent.com/75434179/178314303-960686ac-82af-4243-9799-bf0c93db11dc.png">

Reply:
Hi @jsug9 try this [solution](https://stackoverflow.com/questions/67371958/rails-routing-how-to-rename-the-params-hash-keys-in-a-nested-resource-block)

Reply:
Thanks 😁